rooting: Update method 3 to use `fastboot`

shymega opened the pull request:
Also change to `boot` partition instead of `boot_a`, in case of slot selection state change.

ZimbiX reviewed (changes requested):
I'm good with the switch to fastboot there for consistency, but this won't currently work, as the steps leading up to that haven't put the device into fastboot mode.

shymega committed:
rooting: Update method 3 to use `fastboot`

Also change to `boot` partition instead of `boot_a`, in case of slot
selection state change.

Signed-off-by: Dom Rodriguez <shymega@shymega.org.uk>
